fees/pinnako.ts

```diff
@@ -0,0 +1,66 @@
+import { Chain } from "@defillama/sdk/build/general";
+import { gql, request } from "graphql-request";
+import type { ChainEndpoints } from "../adapters/types";
+import { Adapter } from "../adapters/types";
+import { ZKSYNC } from "../helpers/chains";
+import { getTimestampAtStartOfDayUTC } from "../utils/date";
+
+const endpoints = {
+    [ZKSYNC]: "https://graph.pinnako.io/subgraphs/name/ekmanss/stats",

```

Comment:
seem tvl is set era chain chain can set `CHAIN.ERA` instead new value

Reply:
Has changed